console.js

```diff
@@ -6,7 +6,107 @@
+            "git-filter-branch: Hide your dirty laundry",
+            "git - control your forest of trees",
+            "git - subversion done right",
+            "This is not a joke, it's a commit.",
+            "We're gits on the moon! We format our patches! But there ain't no stashes so we code bad branches and rebase our commits!",
+            "git-cherry: the healthier snack",
+            "Be careful you don't get lost in gitland!",
+            "diff in hell, SVNites!",
+            "gitcoin: REWRITE A COMMIT, GO TO JAIL",
+            "git off!",
+            "Those who cannot --graph their branch are condemned to rebase it",
+            "Once you go git, you never go back.",
+            "This is not the greatest commit in the world, this is just a cherry-pick.",
+            "You can store your code in it!",
+            "Hadn't been for merge-eyed Joe, Ida been commited long time ago",
+            "git commit crim… PUT DOWN THAT GUN AND HANDS UP IN THE AIR!",
+            "No, I'm gitticus!",
+            "git-blame: ruining friendships since 2006",
+            "GitMan: Able to merge 8-headed octopoii in a single command! X-Ray vision of all code history!",
+            "In Git we trust!",
+            "all your rebase are belong to us",
+            "git request-pull: only legal in Nevada.",
+            "git: where we dropped our trunks and all have masters",
+            "git: history is written by the committers.",
+            "Git gets easier once you get the basic idea that branches are homeomorphic endofunctors mapping submanifolds of a Hilbert space.",
+            "Ceci n'est pas une commit",
+            "git out of da choppah!",
+            "DaftPunk.git: clone it, fetch it, merge it, push it; pull it, log it, cherry-pick it",
+            "Non nobis Domine, sed Git da gloriam!",
+            "Updates were rejected: use the --force",
+            "I git by with a little help from my friends",
+            "iGit 2.5 S: THE NEW GIT | MORE PERSONAL | MORE HIPSTER | THE MOST GIT SINCE GIT",
+            "Commits are not objects of love, and should not be judged by their size.",
+            "Always make picking as sweet as the cherries, with gitvia.",
+            "where do SVN-Users keep their HEAD? ... in the trunk!",
+            "Git can remember it for you wholesale",
+            "Git knows what you did last summer!",
+            "A commit a day keeps the mobs away",
+            "git-outhouse: when even plumbing commands are too nice",
+            "May the --force be with you",
+            "War is peace, freedom is slavery, ignorance is strength, svn is version control",
+            "Subversion to crimestop thoughtcrime and unperson Emmanuel Gitstein",
+            "The git-git-slide is fun to dance to, until it isn't",
+            "In case of fire: git commit, git push, leave the building",
+            "Don't forgit to bring a towel",
+        ]
+        console.log(`${jokes.random()}`)
```

Comment:
Also add a semi-colon here.

Reply:
Yes, @sajoy I sourced it from a github repository. I will update in Readme.
I will add the semi-colon too.

I have a doubt, How do you set up linter config for a single Js file? Only if this is converted into a project, we can config lint right?